### PR TITLE
Enable Catalan language

### LINKED ISSUE #3022: Please, add catalan translation
Hi,

In the next version, can you include the Catalan language version, which is already 100% translated?

https://translate.wordpress.com/projects/pocket-casts/android/ca/default/

### PR DESCRIPTION
**Pull request by MiSikora**
## Description

This enables the Catalan language.

You can see that app is more or less fully translated here: https://translate.wordpress.com/projects/pocket-casts/android/ca/default

Closes #3022

## Testing Instructions

1. Change your system language to Catalan.
2. Verify that the app supports it.

## Checklist
- [x] If this is a user-facing change, I have added an entry in CHANGELOG.md
- [x] ~Ensure the linter passes (`./gradlew spotlessApply` to automatically apply formatting/linting)~
- [x] ~I have considered whether it makes sense to add tests for my changes~
- [x] All strings that need to be localized are in `modules/services/localization/src/main/res/values/strings.xml`
- [x] ~Any jetpack compose components I added or changed are covered by compose previews~
- [x] ~I have updated (or requested that someone edit) [the spreadsheet](https://docs.google.com/spreadsheets/d/107jqrutZhU0fVZJ19SBqxxVKbV2NWSdQC9MFYdLiAxc/edit?usp=sharing) to reflect any new or changed analytics.~

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR is larger than 500 lines of changes. Please consider splitting it into smaller PRs for easier and faster reviews.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by MiSikora** [comment]
I put the PR on hold until this is clarified: p1729592526560929-slack-C02AED43D

File: fastlane/Fastfile [hunk 1]
```diff
@@ -34,23 +34,26 @@ BUILD_CODE_FORMATTER = Fastlane::Wpmreleasetoolkit::Versioning::SimpleBuildCodeF
...
-  { glotpress: 'ja', android: 'ja', google_play: 'ja-JP',  promo_config: {} },
-  { glotpress: 'ko', android: 'ko', google_play: 'ko-KR',  promo_config: {} },
-  { glotpress: 'nl', android: 'nl', google_play: 'nl-NL',  promo_config: {} },
-  { glotpress: 'nb', android: 'nb', google_play: 'nb-NB',  promo_config: {} },
-  { glotpress: 'pt-br', android: 'pt-rBR', google_play: 'pt-BR', promo_config: {} },
-  { glotpress: 'ru', android: 'ru', google_play: 'ru-RU',  promo_config: {} },
-  { glotpress: 'sv', android: 'sv', google_play: 'sv-SE',  promo_config: {} },
-  { glotpress: 'zh-cn', android: 'zh', google_play: 'zh-CN', promo_config: {} },
-  { glotpress: 'zh-tw', android: 'zh-rTW', google_play: 'zh-TW', promo_config: {} }
+  # There is an option argument 'google_play' that can be used for release notes.
```
Comment (MiSikora) [inline fastlane/Fastfile:37]:
```suggestion
  # There is an optional argument 'google_play' that can be used for release notes.
```

File: modules/services/localization/src/main/res/values/available_languages.xml [hunk 2]
```diff
@@ -3,22 +3,6 @@
...
-    <item>ja</item>
-    <item>ko</item>
-    <item>nl</item>
-    <item>nb</item>
-    <item>pt_BR</item>
-    <item>ru</item>
-    <item>sv</item>
-    <item>zh</item>
-    <item>zh_TW</item>
+    <item>ca</item>
```
Comment (MiSikora) [inline modules/services/localization/src/main/res/values/available_languages.xml:6]:
```suggestion
    <item>ca</item>
    <item>de</item>
    <item>es</item>
    <item>es_MX</item>
    <item>en_GB</item>
    <item>fr</item>
    <item>fr_CA</item>
    <item>it</item>
    <item>ja</item>
    <item>ko</item>
    <item>nl</item>
    <item>nb</item>
    <item>pt_BR</item>
    <item>ru</item>
    <item>sv</item>
    <item>zh</item>
    <item>zh_TW</item>
```

File: modules/services/localization/src/main/res/values/available_languages.xml [hunk 3]
```diff
@@ -3,7 +3,7 @@
 <resources xmlns:tools="http://schemas.android.com/tools">
   <string-array name="available_languages" translatable="false" tools:ignore="InconsistentArrays">
     <item>en_US</item>
-    <item>ar</item>
+    <item>ca</item>
```
Comment (MiSikora) [inline modules/services/localization/src/main/res/values/available_languages.xml:6]:
```suggestion
    <item>ar</item>
    <item>ca</item>
```

**Pull request merged**